src/js/components/bulkDownload/awards/AwardDataContent.jsx

```diff
@@ -173,7 +173,7 @@ export default class AwardDataContent extends React.Component {
                     <div className="download-info__section">
                         <div className="download-info__section-heading">What is award data?</div>
                         <p>
-                            Award data contains all the details of our prime award and sub-award records.
+                            Award data contains all the details of our prime award and Sub-Awards records.
```

Comment:
This can stay as-is because we are not going to title case when used in a sentence, and it can stay singular.